cmd/build.go

```diff
@@ -55,46 +48,52 @@ func Build(ctx context.Context) *cobra.Command {
 			if err := utils.CheckIfDirectory(path); err != nil {
 				return fmt.Errorf("invalid build context: %s", err.Error())
 			}
+			options.Path = path
 
-			if file == "" {
-				file = filepath.Join(path, "Dockerfile")
+			if options.File == "" {
+				options.File = filepath.Join(path, "Dockerfile")
 			}
 
-			if err := utils.CheckIfRegularFile(file); err != nil {
+			if err := utils.CheckIfRegularFile(options.File); err != nil {
 				return fmt.Errorf("invalid Dockerfile: %s", err.Error())
 			}
 
 			buildKitHost, isOktetoCluster, err := build.GetBuildKitHost()
 			if err != nil {
 				return err
 			}
-			log.Information("Running your build in %s...", buildKitHost)
+			if buildKitHost == "" || options.BuildLocally {
+				log.Information("Running your build in docker...")
+			} else {
+				log.Information("Running your build in %s...", buildKitHost)
```

Comment:
let's only log the non-default option to keep it less disruptive.

Reply:
This is to show the user where it will be build on blue: 
<img width="578" alt="Captura de pantalla 2021-09-01 a las 18 33 57" src="https://user-images.githubusercontent.com/25170843/131709276-d94c7b0f-1841-4467-9490-d8d1317b09b4.png">

Reply:
ah cool 👍 , nevermind then.